src/styles/globals.css

```diff
@@ -16,3 +16,7 @@
 	background-image: linear-gradient(to right, #1c7ed6, #4dabf7, #339af0, #1c7ed6);
 	animation: gradient-text-animation 4s infinite linear;
 }
+
+.mantine-4ptjjz {
```

Comment:
Please refer to the [styles API](https://mantine.dev/styles/styles-api/). I'm assuming you used inspect element and grabbed the class name from the dom, but this style name will change sporadically, it's essentially randomly generated. It could be changed on package update, on prod build, god knows